app/views/finance/financial_transactions/_transactions.html.haml

```diff
@@ -40,3 +43,8 @@
           %td.numeric{style: 'width:5em'}
             - if t.financial_transaction_type.financial_transaction_class == c
               = format_currency t.amount
+        - if with_hidden
+          %td
+            - unless t.hidden?
+              = link_to t('ui.delete'), finance_ordergroup_transaction_path(t.ordergroup, t), method: :delete,
+                data: {confirm: t('ui.confirm_delete', name: t.note)}, class: 'btn btn-danger btn-mini'
```

Comment:
Great to only show the option to revert a transaction when hidden ones are shown.

Shouldn't we call this button _revert_ instead of _delete_?

Reply:
I'd like to convey to the user that one really can't 'delete' transactions, but one can undo their effects. Perhaps it's not that common to have a single word for that, like _revert_ or _reverse_. In Dutch it would be _terugdraaien_. Or use an undo icon, perhaps (perhaps with a tooltip, but there's also a confirmation dialog that would explain it)?

Reply:
i vote for "undo" - it is bad english, but a known computer term.  revert is technically better, but users aren't all technical.

Reply:
i like the idea of an icon, which allows us to use a longer text: "Revert" (en), "Rückgängig machen" (de) "Terugdraaien" (nl)

Reply:
i'v changed it to an icon, with a descriptive tooltip